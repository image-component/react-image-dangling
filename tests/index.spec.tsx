/* eslint-disable comma-dangle */
import React from 'react';
import { mount } from 'enzyme';
import Temp from '../src';

describe('temp', () => {
  it('render', () => {
    const wrapper = mount(<Temp />);
    expect(wrapper).toMatchSnapshot();
  });

  it('render text', () => {
    const wrapper = mount(<Temp msg="test22" />);
    expect(wrapper.find('h2').children().props().children).toEqual('test22');
  });

  it('render classname', () => {
    const wrapper = mount(<Temp className="test33" />);
    expect(wrapper.find('.test33').length).toBeTruthy();
  });
});
