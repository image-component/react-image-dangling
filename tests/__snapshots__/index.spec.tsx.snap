// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`temp render 1`] = `
<Temp>
  <h2>
    <div>
      HI
    </div>
  </h2>
</Temp>
`;
